frontend/src/components/GovtMasthead/GovtMasthead.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react'
+import { BiChevronDown, BiChevronUp } from 'react-icons/bi'
+import {
+  Box,
+  chakra,
+  Collapse,
+  Flex,
+  Icon,
+  Stack,
+  Text,
+  useDisclosure,
+} from '@chakra-ui/react'
+
+import { BxsBank } from '~assets/icons/BxsBank'
+import { BxsLockAlt } from '~assets/icons/BxsLockAlt'
+import Link from '~components/Link'
+
+import { GovtMastheadIcon } from './GovtMastheadIcon'
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface GovtMastheadProps {}
+
+export const GovtMasthead = ({}: GovtMastheadProps): JSX.Element => {
+  const { isOpen, onToggle } = useDisclosure()
+
+  return (
+    <Box>
+      <chakra.button
+        bg="neutral.200"
+        py={{ base: '0.5rem', md: '0.375rem' }}
+        px={{ base: '1.5rem', md: '1.75rem', lg: '2rem' }}
+        textStyle={{ base: 'legal', md: 'caption-2' }}
+        onClick={onToggle}
+        display="flex"
+        width="100%"
+      >
+        <GovtMastheadIcon
+          fontSize="1rem"
+          mr={{ base: '0.25rem', lg: '0.5rem' }}
+          my={{ base: '0px', md: '2px' }}
+        />
+        <Flex alignItems="center" flexWrap="wrap">
+          <Text my="2px">A Singapore government agency website.&nbsp;</Text>
+          <Link tabIndex={0}>
+            How to identify{' '}
+            {isOpen ? (
+              <Icon
+                as={BiChevronUp}
+                fontSize={{ base: '1rem', md: '1.25rem' }}
+              />
+            ) : (
+              <Icon
+                as={BiChevronDown}
+                fontSize={{ base: '1rem', md: '1.25rem' }}
+              />
```

Comment:
seems to be able to be simplified by changing the `as` prop conditionally:

```tsx
<Icon
  as={isOpen ? BiChevronUp : BiChevronDown}
  ...
/>
```

Reply:
done in 64c4351 thank you~